app/players/elves/druid.py

```diff
@@ -0,0 +1,19 @@
+from app.players.elves.elf import Elf
+
+
+class Druid(Elf):
+    def __init__(
+            self,
+            nickname: str,
+            musical_instrument: str,
+            favourite_spell: str) -> None:
+        super().__init__(nickname, musical_instrument)
+        self.favourite_spell = favourite_spell
+
+    def get_rating(self) -> int:
+        return len(self.favourite_spell)
+
+    def player_info(self) -> str:
+        return (f"Druid {self.nickname}. "
+                f"{self.nickname} has a favourite spell: "
+                f"{self.favourite_spell}")
```

Comment:
Ensure that the `player_info` method accesses the `_favourite_spell` attribute if you change its name to be protected.